helm/aws-load-balancer-controller/templates/deployment.yaml

```diff
@@ -162,6 +162,9 @@ spec:
         {{- if .Values.certDiscovery.allowedCertificateAuthorityARNs }}
         - --allowed-certificate-authority-arns={{ .Values.certDiscovery.allowedCertificateAuthorityARNs }}
         {{- end }}
+        {{- if .Values.loadBalancerClass }}
+        - --defaultLoadBalancerClass={{ .Values.loadBalancerClass }}
```

Comment:
we do not have `defaultLoadBalancerClass` as controller flag, do you mean `- --load-balancer-class` here?
see the supported configurations here: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.7/deploy/configurations/

Reply:
sry
--load-balancer class is correct
I did not commit after changing the code locally during the test.

I'll fix it soon